code/modules/uplink/uplink_items.dm

```diff
@@ -1166,7 +1166,7 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	item = /obj/item/clothing/shoes/chameleon/noslip
 	cost = 3
 	exclude_modes = list(/datum/game_mode/nuclear, /datum/game_mode/nuclear/clown_ops)
-	player_minimum = 20
+	player_minimum = 15
```

Comment:
Honestly should just be removed imo

Reply:
can really see the no slip shoes being used to murder the entire crew on lowpop kek

Reply:
They are barely even used, why not make them more available?

Reply:
I would remove it but it's already been put up as a vote so eh